doc/thing/supported-boards/knot-dk.rst

```diff
@@ -0,0 +1,140 @@
+| SW1   | Button 1   | Factory Reset | P0.11 / P1.07 | Reserved   |
++-------+------------+---------------+---------------+------------+
+| SW2   | Button 2   | Application   | P0.12 / P1.07 | Available  |
++-------+------------+---------------+---------------+------------+
+| SW3   | Button 3   | Application   | P0.24         | Available  |
++-------+------------+---------------+---------------+------------+
+| SW4   | Button 4   | Application   | P0.25         | Available  |
++-------+------------+---------------+---------------+------------+
+| SW5   | Boot/Reset | Reboot        |               | Reserved   |
++-------+------------+---------------+---------------+------------+
+
+LEDs
+''''
+
++-------+-------------+-------+-------+---------------+------------+
+| Part  | Description | Color | Pin   | Solder bridge | Permission |
++=======+=============+=======+=======+===============+============+
+| LED 1 | Status      | Green | P0.13 | SB5           | Reserved   |
++-------+-------------+-------+-------+---------------+------------+
+| LED 2 | Application | Green | P0.14 | SB6           | Available  |
++-------+-------------+-------+-------+---------------+------------+
+| LED 3 | Application | Green | P0.15 | SB7           | Available  |
++-------+-------------+-------+-------+---------------+------------+
+| LED 4 | Application | Green | P0.16 | SB8           | Available  |
++-------+-------------+-------+-------+---------------+------------+
+
+Analog Input
+''''''''''''
+
++-------+----------------+-------+------------+
+| Name  | Arduino Naming | Pin   | Permission |
++=======+================+=======+============+
+| AIN 0 | Unavailable    | P0.02 | Available  |
++-------+----------------+-------+------------+
+| AIN 1 | A0             | P0.03 | Available  |
++-------+----------------+-------+------------+
+| AIN 2 | A1             | P0.04 | Available  |
++-------+----------------+-------+------------+
+| AIN 3 | Unavailable    | P0.05 | Available  |
++-------+----------------+-------+------------+
+| AIN 4 | A3             | P0.28 | Available  |
++-------+----------------+-------+------------+
+| AIN 5 | A4             | P0.29 | Available  |
++-------+----------------+-------+------------+
+| AIN 6 | A5             | P0.30 | Available  |
++-------+----------------+-------+------------+
+| AIN 7 | A5             | P0.31 | Available  |
++-------+----------------+-------+------------+
+
+-------------------------------------------------------------------------------
+
+Flash instructions
+------------------
+
+To flash the KNoT DK connect a USB cable between KNoT DK and a PC using the J2
+connector and turn the power switch on. Make sure the SW9 switch is on VDD and the
+SW6 switch is on default position. Use nRF Connect or KNoT CLI to flash the
+image on your board.
+
+.. figure:: ../../../_static/dk_flash.png
+   :scale: 50 %
+   :alt: DK - nRF52840 (PCA10056)
+   :align: center
+
+   Switches position when flashing KNoT DK
```

Comment:
I would use `Position of switches to flash KNoT DK`. That would avoid the user to mistake it with "The position is switched when flashing KNoT DK".